test/idl/consistency.js

```diff
@@ -33,7 +33,9 @@ const ignorableAnomalies = [
   'incompatiblePartialIdlExposure',
   'singleEnumValue',
   'unexpectedEventHandler',
-  'wrongCaseEnumValue'
+  'wrongCaseEnumValue',
+  'noEvent',
+  'urlType'
```

Comment:
I'm not convinced we should add these to the guarantees of webref - this will make the cost of patching higher, for no obvious benefit

Reply:
ha! sorry for the noise then :)